src/SubdomainInput/SubdomainInput.jsx

```diff
@@ -46,6 +54,35 @@ export const SubdomainInput = forwardRef(
     ref
   ) => {
     const inputRef = useRef()
+    const totalWidth = useRef()
+    const [maxInputWidth, setMaxInputWidth] = useState()
+
+    const prefixWidth = useMemo(
+      () =>
+        getTextWidth(
+          prefix,
+          `${theme.fontWeights.lighter} ${theme.fontSizes[2]} ${theme.fonts.primary}`
```

Comment:
what do you think if we extract `${theme.fontWeights.lighter} ${theme.fontSizes[2]} ${theme.fonts.primary}` to a method ?

Reply:
I think it's a good idea 👍🏻